tools/scripts/fuzz.sh

```diff
@@ -27,12 +27,16 @@ mkdir -p fuzz_corpus
 find test/testdata -iname "*.rb" | grep -v disable | xargs -n 1 -I % cp % fuzz_corpus
 mkdir -p fuzz_crashers/original
 
-echo "running"
 # use top 10 frames to tell different errors apart
 export ASAN_OPTIONS="dedup_token_length=10"
+# set FUZZ_ARGS to a space-delimited list of other args to pass to the fuzzer, for instance
+# --stress-incremental-resolver. by default it is '--'.
+FUZZ_ARGS="${FUZZ_ARGS:---}"
+
+echo "running with FUZZ_ARGS: $FUZZ_ARGS"
 nice "./bazel-bin/test/fuzz/$what" \
-  -use_value_profile=1 \
   -only_ascii=1 \
   -dict=test/fuzz/ruby.dict \
   -artifact_prefix=fuzz_crashers/original/ \
+  "$FUZZ_ARGS" \
```

Comment:
Does this work with multiple args? I'm never right on how variable expansion works in bash with double quotes.

Reply:
ah, I believe it won't. (now that I think of it, I think that's why I wanted to not have this in quotes in the first place.)

Reply:
(in this case, even if FUZZ_ARGS has strings, it'll be parsed as one argument here because we put quotes around it.)

Reply:
Yeah, I know there's a special exception for `"$@"`, and that always makes me second guess myself :)

Reply:
yep, I took advantage of the special case for `"$@"` to try to fix this.